metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -1315,13 +1315,15 @@ def create_dbt_platform_mces(
             self.config.tag_prefix,
             "SOURCE_CONTROL",
             self.config.strip_user_ids_from_email,
+            match_nested_props=True,
```

Comment:
what's the impact of this? 
set `match_nested_props=True` suggests we are making this more restrictive